tests/validator/validator.cpp

```diff
@@ -1112,7 +1127,7 @@ TEST(Validator, validMathCnElementsMissingCellMLNamespace)
     m->addComponent(c);
 
     v.validateModel(m);
-    EXPECT_EQ(size_t(0), v.errorCount());
+    EXPECT_EQ_ERRORS(expectedErrors, v);
```

Comment:
This is an important change from previous validation.  Where there were no errors now there are a lot.  This is because the CellML namespace has not been bought through from the enclosing model.